scripts/dockcross-manylinux-set-vars.sh

```diff
@@ -39,11 +39,11 @@ TARGET_ARCH=${TARGET_ARCH:=x64}
 
 # Specialized manylinux image tag to use for building.
 if [[ ${MANYLINUX_VERSION} == _2_28 && ${TARGET_ARCH} == x64 ]]; then
-  IMAGE_TAG=${IMAGE_TAG:=20230106-1aeaea0}
+  IMAGE_TAG=${IMAGE_TAG:=20230926-9eb419c}
 elif [[ ${MANYLINUX_VERSION} == _2_28 && ${TARGET_ARCH} == aarch64 ]]; then
   IMAGE_TAG=${IMAGE_TAG:=2022-11-19-1b19e81}
```

Comment:
Should this one be updated as well ?

Reply:
This one is currently separate as we work towards arm cross-compilation.